### PR TITLE
feature/103-added example.env

### PR DESCRIPTION
**Pull request by WinterBoii**

### CODE REVIEW

**Review by vikjoh1 — APPROVED** [review]
Looks good!

File: example.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+APP_NAME = SampleApp // Change it to your liking, eg. Flexicharge
```
Comment (eejebring) [inline example.env:1]:
```suggestion
APP_NAME=SampleApp# Change it to your liking, eg. Flexicharge
```

File: example.env [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+REACT_APP_DEV_PORT = 1234 // Port Nr. eg. 8080
```
Comment (eejebring) [inline example.env:2]:
```suggestion
REACT_APP_DEV_PORT=1234# Port Nr. eg. 8080
```

File: example.env [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+REACT_APP_FLEXICHARGE_API_URL = http://example.com // Replace with the API url
```
Comment (eejebring) [inline example.env:3]:
```suggestion
REACT_APP_FLEXICHARGE_API_URL=http://example.com# Replace with the API url
```

**Review by eejebring — CHANGES_REQUESTED** [review]
.env files usualy skip spaces
and comments are made with # not //

**Pull request merged**